packages/contracts-bedrock/src/universal/FeeVault.sol

```diff
@@ -38,45 +21,20 @@ abstract contract FeeVault {
     /// @param from  Address that triggered the withdrawal.
     event Withdrawal(uint256 value, address to, address from);
 
```

Comment:
Why are you changing the fee vault? it doesn't need a diff?

Reply:
modifying this is going to require a lot of coordination between teams, there is no mention of this change in the specs: https://github.com/ethereum-optimism/specs/pull/115/

Reply:
This is an artefact of me experimenting, I can remove these changes when I get back to this.

Reply:
Ok! Thank you for explaining. Just didn't see any specs work for this particular part of the diff so it confused me